.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+'on':
+  push:
+    branches:
+      - master
+      - "dependabot/**"
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: bahmutov/npm-install@v1
```

Comment:
cool, I didn't know about this one :)

Reply:
when in doubt, its often worth checking to see if gleb has a solution :) very often pays off